### ISSUE TITLE
Need example of when a <section> might NOT have a heading

### ISSUE THREAD

**Issue by jonkri** [mdn/sprints#1515]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details
I think it would be useful to clarify what is meant by:

<blockquote>Typically, but not always, sections have a heading.</blockquote>

Specifically, I would like for the article to clarify when a section might *not* have a heading.

(I'm wondering about this myself.)

A [usage note](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section#Usage_notes) specifies:

<blockquote>Each <code>&lt;section&gt;</code> should be identified, typically by including a heading (<code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> element) as a child of the &lt;section&gt; element.</blockquote>

HTML Living Standard specifies:

<blockquote>A section, in this context, is a thematic grouping of content, typically with a heading.</blockquote>

The section examples at the [<code>section</code> MDN page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) and https://html.spec.whatwg.org/multipage/sections.html#the-section-element all use <code>&lt;h1&gt;</code>-<code>&lt;h6&gt;</code> elements.

**Comment by jonkri** [comment]
@chrisdavidmills: Hi! I don't see a clarification in the MDN docs. Is this a "wontfix"? Thanks!

**Comment by chrisdavidmills** [comment]
@jonkri Yeah, sorry for the lack of explanation here. I'm trying to clear out some old bugs that we never ended up getting round to fixing, and I'm not sure when we will get around to with the current backlog. This happened to be on that list.

Is this something you'd have an interest in fixing?

**Comment by jonkri** [comment]
@chrisdavidmills:

Thanks for the reply!

The reason I created this issue is that I, based on the available documentation, haven't been able to figure out under what circumstances headings are actually optional.

I have noticed that some developers are using `section` elements pretty much as `div` elements, like for layout. I have always assumed that this is wrong semantically, but the docs are not clear on the issue. I thought that clarifying the issue would help against this potential misuse of the element.

**Comment by jonkri** [comment]
Also, could not unfixed issues be left open, even if it's uncertain when and if they can be dealt with?

**Comment by jonkri** [comment]
(I guess it should be closed if it's a "wontfix", though, but in that case, perhaps that could be explicitly mentioned when the issue is closed, so that people don't have to be left wondering if it's resolved or not.)

**Comment by chrisdavidmills** [comment]
Tell you what. I'll reopen it and move it to our `content` repo, where content bugs are now reported.

Then I might well have a go at fixing it myself. I've always had a bugbear about sectioning element usage, as even the spec seems kind of ambiguous, but I do have some good idea on when these should be used.
